lib/rubygems/commands/update_command.rb

```diff
@@ -306,4 +315,10 @@ def which_to_update(highest_installed_gems, gem_names, system = false)
     result
   end
 
+  private
+
+  def oldest_supported_version
+    @oldest_supported_version ||= Gem::Version.new("2.5.2")
```

Comment:
Can you add the comment about the Ruby version? It's hard to understand the Ruby version from RubyGems version.

Reply:
Yes, I will, good point :+1:.

Reply:
Maybe instead of a comment put it in a constant?

Reply:
What should I put in a constant?

Reply:
Oh, I see. Yiu mean something like `RUBYGEMS_VERSION_PROVIDED_BY_RUBY_2_3` or something like that?

Reply:
Yes

Reply:
`RUBYGEMS_VERSION_PROVIDED_BY_OLDEST_RUBY_SUPPORTED`? :shipit: